.github/workflows/testdata/event-release.json

```diff
@@ -1,15 +1,15 @@
 {
```

Comment:
Is this file used anywhere other than in the example `act` command in the README? Is there a requirement that this is kept up to date with our latest release? If so, is there someway to externalize this state? Or have it autogenerated at use time?

Reply:
This is a fair comment. There is no requirement for this to be kept up to date. It could totally be generated at use. I had thought about it but the cost/benefit hasn't pushed me over the edge to go through that effort.